pkg/config/config_test.go

```diff
@@ -556,6 +556,135 @@ func TestConfig_GetPackages(t *testing.T) {
 	}
 }
 
+func TestConfig_ShouldGenerateInterface(t *testing.T) {
```

Comment:
Thanks for adding this test. I had integration tests that were checking this did the right thing but good to have explicit unit test as well 👍🏻